apps/beeai-ui/src/hooks/useToTopButton.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2025 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use client';
+
+import { noop } from '#utils/helpers.ts';
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+interface Props {
+  enabled?: boolean;
+}
+
+export function useToTopButton({ enabled = true }: Props = {}) {
+  const ref = useRef<HTMLDivElement>(null);
+  const [showButton, setShowButton] = useState(false);
+
+  const handleScroll = useCallback((event: Event) => {
+    const { scrollTop } = event.target as HTMLDivElement;
+
+    setShowButton(scrollTop > SCROLLED_OFFSET);
```

Comment:
`SCROLLED_OFFSET` is offset, that's true, but in the grand scheme of things, it means something else. Why is it 48?
How about calling it `APP_HEADER_HEIGHT`?